### PR TITLE
Pay bills

### PR DESCRIPTION
**Pull request by RemusCM**
#41 Done.. Bills should already exist.

That many lines cause my other pr wasnt merged yet

### CODE REVIEW

**Comment by samantha-kerr** [comment]
@RemusCM can you pull master, there are changes from your previous PR that show in here

File: bills.php [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+<head>
+
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+
+    <title>Pay your bills.</title>
+</head>
+<body>
+
+
+<form method="post" action="index.php" id="pay-bills" name="pay-bills">
```
Comment (samantha-kerr) [inline bills.php]: wrap this in the same div definition used on other pages

File: bills.php [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+    <h1>Pay your bills</h1>
```
Comment (samantha-kerr) [inline bills.php]: h4

File: bills.php [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+                }
+
+                ?>
+
+            </div>
+
+
+
+    </div>
+
```
Comment (samantha-kerr) [inline bills.php]: Wyy all these extra spaces

Comment (RemusCM) [reply bills.php]: I'm claustrophobic

**Pull request merged**